db/migrate/20190219090835_create_products.rb

```diff
@@ -0,0 +1,11 @@
+class CreateProducts < ActiveRecord::Migration[5.2]
+  def change
+    create_table :products do |t|
+      t.string :product_name
```

Comment:
Do you have any reason why you don't just call this `name`?

Reply:
We might have another table with name later on. So this is just more clear.

Reply:
The models are independent of each other, so there is no need to give each model unique attribute names. When calling on the products name it will look silly (e.g `Product.product_name`)

Reply:
fixed